### DISCUSSION TITLE
smtp > localhost

### DISCUSSION THREAD

**Discussion by waran70** [YetiForceCompany/YetiForceCRM#14536] [How to] [answered]
Can you gents let me know how to configure smtp so it uses localhost? My postfix  is nicely configured but I believe I used all of combination of settings while it still don't like it.
Maybe some tip out there?

**Comment by mariuszkrzaczkowski** [comment]
what do you have in logs?
try with php.ini
![image](https://user-images.githubusercontent.com/10198654/114344741-aad23280-9b60-11eb-814f-6159bd8db320.png)

**Comment by apcloic** [comment, marked as answer]
@waran70 
I'm using postfix/localhost, basic configuration, works perfectly. here's my settings.
![image](https://user-images.githubusercontent.com/2930560/114703722-ef5afb00-9d25-11eb-8801-14e5981e3171.png)
